.idea/compiler.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
Not sure what all this stuff is but please delete & ignore all of `.idea/`.

Reply:
Yes, it is for general code style and cleanup and extending for http to https redirect